File: 1151 - Fibonacci Fácil.go
package main

import "fmt"

func main() {
	var number int
	fmt.Scanln(&number)
	x := 0
	z := 1
	y := 0
	for i := 0; i < number; i++ {
		if i != number-1 {
			fmt.Printf("%d ", x)
		} else {
			fmt.Printf("%d", x)
		}
		x = z
		z += y
		y = x
	}
	fmt.Print("\n")
}

File: 1151 - Fibonacci Fácil.py
number = int(input())
x = 0
z = 1
y = 0
for i in range(number):
    if i < number - 1:
        print("%d" % x ,end=' ')
    else:
        print("%d" % x, end='\n')
    x = z
    z += y
    y = x
